demos/ci-cd-ops-role-sdp/README.md

```diff
@@ -0,0 +1,30 @@
+# Enhance CICD Demo to Add the "OPs" side of the story - Trello card # 132
```

Comment:
@themoosman i think this README page, rather than calling out what Trello task is completed, should just be a brief overview of what this Demo does, and a description of all of the various bits of content you've included. Assume the audience knows nothing about our trello board, or task assignments. Could be a red hatter... could be a community member.